http/oidc/src/main/java/org/wildfly/security/http/oidc/RefreshableOidcSecurityContext.java

```diff
@@ -151,7 +151,7 @@ public boolean refreshToken(boolean checkActive) {
             IDToken idToken;
             try {
                 TokenValidator tokenValidator = TokenValidator.builder(clientConfiguration).build();
-                TokenValidator.VerifiedTokens verifiedTokens = tokenValidator.parseAndVerifyToken(idTokenString, accessTokenString, cookie);
+                TokenValidator.VerifiedTokens verifiedTokens = tokenValidator.parseAndVerifyToken(idTokenString, accessTokenString, cookie, false);
```

Comment:
With the approach I mentioned below, we could leave this unchanged (and the nonce won't get validated).